File: Assignments/Assignment_12/p6.c
#include<stdio.h>
void main(){

        int row;
        printf("Enter the no of rows : \n");
        scanf("%d",&row);
        for(int i=0;i<row;i++){

                for(int j=0;j<row;j++){

                        if(row%2==0){

                                if(j%2==0){

                                        printf("%d\t",(65+j));
                                }else{

                                        printf("%c\t",(65+j));
                                }
                        }else{

                                printf("%c\t",(65+j));
                        }
                }
                printf("\n\n");
        }
}

File: JAVA/11.Triangle_Pattern/p6.java
import java.util.*;

class Demo{

	public static void main(String[] args){
	
		Scanner sc = new Scanner(System.in);
		System.out.println("Enter no. of rows: ");
		int row = sc.nextInt();
	
		for(int i=row;i>0;i--){
		
			for(int j=0;j<i;j++){
			
					System.out.print(i+"\t");
			}System.out.println();
		}
	}
}
